test/classes/eik-config.test.js

```diff
@@ -5,77 +5,156 @@ const EikConfig = require('../../lib/classes/eik-config');
 test('basic config properties', (t) => {
     const config = new EikConfig({
         name: 'pizza',
-        files: 'secret receipe',
-        version: 'deep dish',
+        server: 'http://server',
+        files: { '/': 'pizza' },
+        version: '0.0.0',
     });
     t.equal(config.name, 'pizza');
-    t.equal(config.files, 'secret receipe');
-    t.equal(config.version, 'deep dish');
+    t.same(config.files, { '/': 'pizza' });
+    t.equal(config.version, '0.0.0');
     t.end();
 });
 
 test('map property', (t) => {
-    let config = new EikConfig({ 'import-map': 'cheese from italy' });
-    t.deepEqual(config.map, ['cheese from italy']);
+    let config = new EikConfig({
+        name: 'pizza',
+        server: 'http://server',
+        files: { '/': 'pizza' },
+        version: '0.0.0',
+        'import-map': 'http://map',
+    });
+    t.deepEqual(config.map, ['http://map']);
 
-    config = new EikConfig({ 'import-map': ['gouda', 'mozzarella'] });
-    t.deepEqual(config.map, ['gouda', 'mozzarella']);
+    config = new EikConfig({
+        name: 'pizza',
+        server: 'http://server',
+        files: { '/': 'pizza' },
+        version: '0.0.0',
```

Comment:
Perhaps it is worth extracting out a VALID_CONFIG object that can be reused to make the tests less noisy?

Reply:
good idea